rundeckapp/grails-app/views/execution/show.gsp

```diff
@@ -292,7 +315,7 @@ search
                               <g:link controller="scheduledExecution"
                                       action="execute"
                                       id="${scheduledExecution.extid}"
-                                      class=" pull-right"
+                                      class="btn btn-default btn-xs pull-right"
```

Comment:
This adds a button for the Run Again trigger, in keeping with the button usage on the job page for the "Run Job Now" Button